lib/screens/StudyModuleScreen.dart

```diff
@@ -44,10 +44,13 @@ class _StudyModuleScreenState extends State<StudyModuleScreen> {
                 }),
               )
             },
-            child: Text(
-              _trimName(eachStudyModule.getTitle()),
-              style: themeData.textTheme.headline4,
-              textAlign: TextAlign.left,
+            child: Container(
+              width: double.infinity, // 100% width of parent
```

Comment:
Is there any reason we don't use context.of here to directly get the parent width?

Reply:
Honestly just because I don't know how that works haha. Could you give me a code snippet so I can change it?

Reply:
Keeping it your way seems very valid too! Check this thread for alternatives
https://stackoverflow.com/questions/41558368/how-can-i-layout-widgets-based-on-the-size-of-the-parent

Reply:
In that case I think I will keep it like this. I think with the comment it is pretty self explanatory what is happening there and to me it seems a bit clearer then fiddling with some sort of widget wrap.